x-pack/plugins/cases/public/components/cases_context/use_cases_features.test.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import { useCasesFeatures, UseCasesFeatures } from './use_cases_features';
+import { TestProviders } from '../../common/mock';
+
+describe('useCasesFeatures', () => {
+  it('returns isAlertsEnabled=true and isSyncAlertsEnabled=true if the feature.alerts={sync: true, enabled: true}', async () => {
```

Comment:
nit: you can reduce all this code test file to a single test if you use table tests.